pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -282,7 +282,7 @@ def _loaded_ts_sorted(self) -> np.ndarray:
     def _fill_gaps(self, timestamps: np.ndarray) -> np.ndarray:
         time_diff = np.diff(timestamps)
         median_time_diff = np.median(time_diff)
-        gap_start_idc = np.flatnonzero(time_diff > 2 * median_time_diff)
+        gap_start_idc = np.flatnonzero(time_diff > 5 * median_time_diff)
```

Comment:
This change would result in only gaps with 4 or more dropped frames being filled, assuming them being filled with `median_time_diff` distance.